plugin/build.gradle

```diff
@@ -217,6 +237,76 @@ dependencies {
     implementation "androidx.core:core-splashscreen:$versions.splashscreenVersion"
 }
 
+task buildOpenxrValidationLayersAndroid {
+    for (abi in ['arm64-v8a', 'x86_64']) {
+        def suffix = (abi == 'arm64-v8a') ? 'Arm64' : (abi == 'x86_64') ? 'X86_64' : abi
+
+        def libraryTargetDir = file("${khronosOpenxrValidationLayersAndroidLibraryTargetDir}/${abi}/${abi}")
```

Comment:
Why does the `libraryTargetDir` need `${abi}` twice as suffix?

Reply:
It's because we are conditionally adding the path for the specific ABIs. So, for example, when we want to include x86_64 binaries, we tell Gradle to use a particular path, ("${khronosOpenxrValidationLayersAndroidLibraryTargetDir}/${abi}") and then it wants that path to also contain a directory named with the ABI.

We're already doing the same thing for the Khronos OpenXR Loader - this is just repeating that for the validation layers.